dts/arm/nxp/nxp_lpc55S6x.dtsi

```diff
@@ -80,6 +80,16 @@
 			gpio-controller;
 			#gpio-cells = <2>;
 		};
+
+		usbd:usbd@40084000 {
+			compatible = "nxp,kinetis-usbd";
```

Comment:
You need a new compatible since this is different hardware from Kinetis

Reply:
Thanks Maureen, I will update it. There are many failed cases in shippable because there are changes in the hal_nxp. How to sync the hal_nxp's PR to this PR and let the Shippable to get the correct codes of hal_nxp?

Reply:
In `west.yml`, set the hal_nxp revision to `pull/9/head`